docroot/sites/cardinalservice/settings/default.local.settings.php

```diff
@@ -158,3 +158,12 @@
 $settings['trusted_host_patterns'] = [
   '^.+$',
 ];
+
+// Include additional settings files for local development
+$additionalSettingsFiles = [__DIR__ . '/settings.ddev.php'];
```

Comment:
Could you use `docroot/sites/settings/local.settings.php` for that? I'm not sure what the contents of the file are.

Reply:
@pookmish It's an autogenerated settings file that Ddev creates on first startup: They optimize it for local dev. It's not tracked. Here's the contents

```
<?php

/**
 * @file
 * This is a Drupal 10 settings.ddev.php file automatically generated by DDEV.
 *
 * DDEV manages this file and may delete or overwrite it unless this
 * comment, marked with #ddev-generated, is removed. It is recommended
 * that you leave this file alone.
 */

$host = "db";
$port = 3306;
$driver = "mysql";

$databases['default']['default']['database'] = "cardinalservice";
$databases['default']['default']['username'] = "db";
$databases['default']['default']['password'] = "db";
$databases['default']['default']['host'] = $host;
$databases['default']['default']['port'] = $port;
$databases['default']['default']['driver'] = $driver;

$settings['hash_salt'] = '0d0f9020aa322e57266a6de7278d2f96712dfaf698393711cdc1e3966135a92c';

// Recommended setting for Drupal 10 only
$settings['state_cache'] = TRUE;

// This will prevent Drupal from setting read-only permissions on sites/default.
$settings['skip_permissions_hardening'] = TRUE;

// This will ensure the site can only be accessed through the intended host
// names. Additional host patterns can be added for custom configurations.
$settings['trusted_host_patterns'] = ['.*'];

// Don't use Symfony's APCLoader. ddev includes APCu; Composer's APCu loader has
// better performance.
$settings['class_loader_auto_detect'] = FALSE;

// Set $settings['config_sync_directory'] if not set in settings.php.
if (empty($settings['config_sync_directory'])) {
  $settings['config_sync_directory'] = 'sites/default/files/sync';
}

// Override drupal/symfony_mailer default config to use Mailpit.
$config['symfony_mailer.settings']['default_transport'] = 'sendmail';
$config['symfony_mailer.mailer_transport.sendmail']['plugin'] = 'smtp';
$config['symfony_mailer.mailer_transport.sendmail']['configuration']['user'] = '';
$config['symfony_mailer.mailer_transport.sendmail']['configuration']['pass'] = '';
$config['symfony_mailer.mailer_transport.sendmail']['configuration']['host'] = 'localhost';
$config['symfony_mailer.mailer_transport.sendmail']['configuration']['port'] = '1025';

// Enable verbose logging for errors.
// https://www.drupal.org/forum/support/post-installation/2018-07-18/enable-drupal-8-backend-errorlogdebugging-mode
$config['system.logging']['error_level'] = 'verbose';
```

Reply:
ah gotcha. thanks for the information :+1:

Reply:
Thanks, Mike! It's nice to have that in the default.local.settings file.